### DISCUSSION TITLE
Grafana Integration with Power BI or Tableau

### DISCUSSION THREAD

**Discussion by pejmanS21** [grafana/grafana#54421] [Dashboards, Panels & Visualizations]
Hi, I was wondering if there is any way to integrate Grafana Dashboard with other tools like **Power BI** or **Tableau**. For example, I want to send `Prometheus metrics` in Grafana panel to `Tableau`.
![Screen Shot 2022-08-30 at 8 23 17 AM](https://user-images.githubusercontent.com/73995528/187345506-8c6435e0-122e-4ce9-b136-cc42a5d0795e.png)

**Comment by Rodrigo-Henrique21** [comment]
@pejmanS21 tem como realizar a importação de um único visual para o grafana a partir de uma imagem docker do grafana. Realizei algumas pesquisas não consegui encontrar como usar o relátorio todo, todavia daria para usar os dados do grafana via API e conectar usando o conector do power bi desktop 

Segue site com um passo a passo do procedimento;

https://www.sqlservercentral.com/articles/power-bi-in-grafana

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
